package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.1.3",
```

Comment:
That's odd. Minimist is a dependency of eslint, so `npm i` should put it in your `node_modules`. I've just deleted my `node_modules` and re-installed, and the tests pass.

However, you're right that all our dependencies should be explicit, so I've added it to the `package.json`.